src/App.tsx

```diff
@@ -1,11 +1,23 @@
-import React from 'react';
-
+import React, {useState} from 'react';
 import { WorkoutInputDialog } from './components/WorkoutInputDialog';
+import WorkoutBoard from './components/WorkoutBoard';
+import AddIcon from '@material-ui/icons/Add';
+import Fab from '@material-ui/core/Fab';
+import {StyledButton} from './styles'
 
 function App() {
+  const [isDialogOpen, setIsDialogOpen] = useState<boolean>(false);
+  const handleClose = () => setIsDialogOpen(false);
   return (
     <>
-      <WorkoutInputDialog isOpen />
+    <WorkoutBoard />
+    <StyledButton >
+      <Fab color="secondary" aria-label="add" onClick={()=>setIsDialogOpen(true)}>
+        <AddIcon />
+      </Fab>
+      </StyledButton>
```

Comment:
I hope you are using prettier plugin in your editor 👀 Lot of indentations are off in this PR.